### ISSUE TITLE
Replicate results with pre-trained model

### ISSUE THREAD

**Issue by VitorGuizilini-TRI** [wvangansbeke/Sparse-Depth-Completion#28]
Hi, I am trying to replicate your results on a different codebase, using the pre-trained model provided, but I am probably doing something wrong (the depth maps are meaningless, even though I can load the weights). My guess is that it's a pre-processing issue, what sort of normalization are you doing on the input RGB and depth information?   Thanks!

**Comment by wvangansbeke** [comment]
Hi @VitorGuizilini-TRI,

It should be fairly easy actually. I don't normalize the depth maps at the input since it results in worse performance (which makes sense actually). The preprocessing is quite limited. The most important thing is to make sure that there is no mismatch between the lidar and rgb frames. I ran the code a few months ago, and got the same numbers of the paper, as did a few other people. 
Hope this helps.

Best,
Wouter
